apps/console/src/extensions/components/subscription/contexts/subscription.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import { TenantTier } from "../models/subscription";
+
+export const SubscriptionContext: React.Context<{
```

Comment:
Import `Context` from react, so we can drop `React` import all together easily when we start using JSX runtime.

Also move the interface to be separate.

Ref: https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/branding/context/branding-preference-context.tsx#L32
```suggestion
export const SubscriptionContext: React.Context<{
```

Reply:
Fixed with 2f9edfa